apps/modelviewer/src/main.tsx

```diff
@@ -48,11 +48,11 @@ const StyledWrapper = styled.div`
 const App = () => {
   return (
     <StyledWrapper>
-      <FusionModelViewer facility="jca" tagsOverlay={mockedTagListA}>
-        <FusionModelViewer.CustomActions>
+      <ModelViewer facility="jca" tagsOverlay={mockedTagListA}>
+        <ModelViewer.CustomActions>
           <Test />
```

Comment:
Lets discuss this app, it seems weird to mix in hardcoded test apps in the apps folder with production apps

Reply:
Lets discuss how to do this.
The test app is very handy when working with the functionality of the model viewer.

Reply:
This app is now removed.